micrometer-core/src/main/java/io/micrometer/core/instrument/MeterRegistry.java

```diff
@@ -846,6 +850,42 @@ public PauseDetector pauseDetector() {
             return pauseDetector;
         }
 
+        /**
+         * Creates and starts a new {@link HighCardinalityTagsDetector} for this registry.
+         * @return This configuration instance.
+         */
+        public Config withHighCardinalityTagsDetector() {
+            highCardinalityTagsDetector = new HighCardinalityTagsDetector(MeterRegistry.this);
```

Comment:
Should we check if a detector is already configured and close that before replacing it? I'm not sure which would be more surprising behavior.

Reply:
👍🏼 fixed